i3pystatus/openvpn.py

```diff
@@ -46,6 +47,7 @@ class OpenVPN(IntervalModule):
         ("status_down", "Symbol to display when down"),
         ("status_up", "Symbol to display when up"),
         ("vpn_name", "Name of VPN"),
+        ("openvpn_lastest", "Use lastest openvpn service names"),
```

Comment:
Sorry I didn't see your reply, yes for the variable name. Either that or  something like `openvpn_latest` would be better I think.